source/faq.txt

```diff
@@ -48,4 +48,25 @@ change tracking, entity-based LINQ operations, and modeling familiar to
 - Intelligent object tracking
 - Entity-based LINQ operations
 - Entity Framework modeling and mapping with the fluent API
-- Automatic database updates through change tracking
\ No newline at end of file
+- Automatic database updates through change tracking
+
+How Can I Manage Indexes with the {+provider-short+}?
+-----------------------------------------------------
+
+Because the {+provider-short+} is built on top of the {+csharp-driver-short+},
+you can manage indexes in your application by using the {+csharp-driver-short+}
+directly. To use driver methods in your {+provider-short+}
+application, call them by using the ``MongoClient`` used to set up your
+``DbContext``.
+
+The following example creates indexes on a collection by using
```

Comment:
S: not necessary, but you specify the sample_mflix.movies collection so you can include that here
```suggestion
The following example creates indexes on the `movies` collection by using
```